src/pybind/rados/setup.py

```diff
@@ -1,51 +1,187 @@
-# Largely taken from
-# https://blog.kevin-brown.com/programming/2014/09/24/combining-autotools-and-setuptools.html
-import os, sys, os.path
+from __future__ import print_function
 
-from setuptools.command.egg_info import egg_info
+import os
+import shutil
+import subprocess
+import sys
+import tempfile
+import textwrap
+from distutils.ccompiler import new_compiler
 from distutils.core import setup
+from distutils.errors import CompileError, LinkError
 from distutils.extension import Extension
-from Cython.Build import cythonize
+from distutils.sysconfig import customize_compiler
 
-def get_version():
+# PEP 440 versioning of the Rados package on PyPI
+# Bump this version, after every changeset
+# NOTE: This version is not the same as get_ceph_version()
+
+__version__ = '0.0.1'
+
+
+def get_ceph_version():
```

Comment:
This `get_ceph_version()` didn't return a PEP 440 compliant version, so this needs discussion. Perhaps it can be a number prefixed by Ceph's version, if that makes sense.